script/deployment/DeployFeeds.s.sol

```diff
@@ -64,6 +64,7 @@ contract DeployFeeds is Script {
                         feedId,
                         configStructured.supportedFeedsData[i].description,
                         uint8(configStructured.supportedFeedsData[i].decimals),
+                        uint8(configStructured.supportedFeedsData[i].decimals),
```

Comment:
same decimals?

Reply:
We should also change config to include input decimals and output decimals

Reply:
yep I know I left this part out to make sure we are all on the same page because jsons can be a pain... will do.